test/user-test.js

```diff
@@ -0,0 +1,147 @@
+import chai from 'chai';
```

Comment:
I already mentioned you should use a single describe. include all the test in one place

Reply:
ok sir